### PR TITLE
Mobile menu with JS

### PR DESCRIPTION
**Pull request by GiuseppeTG**
In this branch, I added some dynamics to the mobile menu using JS matching the Figma template number 5.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,24 +14,32 @@
...
+        </li>
+        <li class="nav-item">
+          <a href="#contact" class="nav-link">Contact</a>
+        </li>
       </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
```
Comment (Wes-Isaac) [inline index.html:42]: - [ ] Great job so far, all the functionalities work just a minor fix, the hamburger menu shouldn't be displayed for desktop devices, better to fix this before going to the next project.

image

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @GiuseppeTG, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are on point ✔️.
- Readme is professional 👀.
- Design is exquisite 😮.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by GiuseppeTG** [comment]
Thank you @Wes-Isaac.

I don't see the hamburger menu on a desktop device. Double checked but when I open it the menu is okay. 

Captura de Pantalla 2022-05-09 a la(s) 12 47 01 p  m

**Review by keroloussamy — APPROVED** [review]
Hi @GiuseppeTG, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by keroloussamy** [comment]
@GiuseppeTG the menu appears when the page is under 1440 width.

**Comment by Wes-Isaac** [comment]
@GiuseppeTG by desktop I mean a device that's not a mobile device(>768px), Please check for those

**Comment by GiuseppeTG** [comment]
@keroloussamy @Wes-Isaac Oh! thanks. I thought desktop was for 1440px. I will fix that (:

**Pull request merged**